common.go

```diff
@@ -81,3 +82,7 @@ var SBTablesOrder = []string{
 	TableEncap,
 	TableSBGlobal,
 }
+
+var LatestSBTablesOrder = []string{
```

Comment:
What is the intent for LatestSBTablesOrder ?TableChassisPrivate should be in SBTablesOrder

Reply:
The intent again is that ChassisPrivate table was only recently added. It doesn't exist on the old releases of OVN.

Reply:
thanks @girishmg for details. So if new table gets added to schema everytime, we keep changing go-ovn with latest* table? I think thats not the end goal. Also our current fork runs against latest master so should be fine. Please rectify if I got it something the other way,